packages/bot-cleaning/src/strategies.ts

```diff
@@ -0,0 +1,47 @@
+import { Market } from "@mangrovedao/mangrove.js";
+import Big from "big.js";
+
+type SnipeParams = {
+  takerWants: Big;
+  takerGives: Big;
+};
+
+/**
+ * This function calculates the minimum assets that a taker is willing to exchange, considering whether it's a bid or ask offer.
```

Comment:
This phrasing is odd: "that a taker is willing to". This is a bot that is making a choice.